### PR TITLE
Issue 107: changing the z-index to account for when the mobile nav is…

### PR DESCRIPTION
**Pull request by ZachStoltz**
… trigger
screen shot 2015-07-06 at 10 11 44 pm
ed

### CODE REVIEW

**Comment by jeffshaver** [comment]
If i go to "My Profile" or "My Initiative" the overlay is on top of the mobile nav. Would be good to fix that in this pr as well :smile:

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@
 .meteor/local
 .meteor/meteorite
 private/local-settings.json
-client/style.sass
```
Comment (jeffshaver) [inline .gitignore:5]: This seem odd. no actual change lol

Comment (ZachStoltz) [inline .gitignore:5]: Yeah, seems odd.  Maybe because I added .idea for Webstorm to the end so the new line was swapped?

**Comment by trustieee** [comment]
Yep, your .idea change was committed then removed, but the file was still different due to newline.

**Pull request merged**